public/cantusdata/helpers/expandr.py

```diff
@@ -86,7 +86,9 @@ class PositionExpander(object):
     position_data_base = None
 
     def __init__(self):
-        self.csv_file = csv.DictReader(open("data_dumps/position_names.csv"))
+        cantusdata_path = os.path.split(os.path.dirname(__file__))[0]
+        public_path = os.path.split(cantusdata_path)[0]
+        self.csv_file = csv.DictReader(open(os.path.join(public_path, "data_dumps", "position_names.csv")))
```

Comment:
If I understand this correctly (i.e., getting the absolute path of django project), this is available through the `settings` file of the project. You could import that path in the `settings.py` of the django project and avoid a redundant call to `os.path.dirname(__file__)`. 

Please check the `settings.py` and let us know if that's an appropriate import for this information.

Reply:
yup. i'll make the fix